packages/chakra-ui/src/ArrayFieldTemplate/ArrayFieldTemplate.tsx

```diff
@@ -0,0 +1,271 @@
+import React, { FunctionComponent } from "react";
+import { utils } from '@rjsf/core';
+
+import { ArrayFieldTemplateProps, IdSchema } from "@rjsf/core";
+
+
+/* RJSF Utils */
+const { isMultiSelect, getDefaultRegistry } = utils;
+
+/* Local Components */
+import AddButton from "../AddButton/AddButton";
+import IconButton from "../IconButton/IconButton";
+import { Divider, FormLabel, Flex, Box } from "@chakra-ui/react";
+
+const ArrayFieldTemplate = (props: ArrayFieldTemplateProps): JSX.Element => {
```

Comment:
In the "Arrays" tab, we shouldn't render the red required asterisks as their own line:

![image](https://user-images.githubusercontent.com/1689183/111927114-ef007480-8a85-11eb-9ec1-544e19e68c24.png)

Here's how it looks like on the default theme:

![image](https://user-images.githubusercontent.com/1689183/111927121-f4f65580-8a85-11eb-9ac3-b7a6db5115bf.png)

On the fluent-ui theme (an alternative, where the asterisks are rendered after the textbox):

![image](https://user-images.githubusercontent.com/1689183/111927128-02abdb00-8a86-11eb-9cb1-85f36aec8dc9.png)